snippets/dart2-only/snippets.json

```diff
@@ -153,8 +153,8 @@
       "",
       "}",
       "",
-      "@Component()",
-      "class ${2:MyComponent}Component extends UiStatefulComponent<${2:MyComponent}Props, ${2:MyComponent}State> {",
+      "@Component2()",
+      "class ${2:MyComponent}Component extends UiStatefulComponent2<${2:MyComponent}Props, ${2:MyComponent}State> {",
       "\t@override",
       "\tMap getDefaultProps() => (newProps());",
       "",
```

Comment:
this applies to all stateful component snippets we should change stateful components to the new convention

instead of 
```
@override
Map getInitialState() => (newState());
```
do
```
  @override
  void init() {
    initializeState(newState());
  }
```

Reply:
Good catch!